### PR TITLE
Make FullTrack.id optional

### LINKED ISSUE #279: `TrackId` in `FullTrack` can be null
**Describe the bug**
According the specs defined in [docs.rs](https://docs.rs/rspotify-model/0.11.2/rspotify_model/track/struct.FullTrack.html), `TrackId` in `FullTrack` is not null. However, [get playlist items](https://developer.spotify.com/documentation/web-api/reference/#/operations/get-playlists-tracks) can return a track object will null id.

**To Reproduce**
1. To get the playlist having a track with null ID, run
```
curl -X "GET" \
"https://api.spotify.com/v1/playlists/2RI4NLi6npcHfCNQQxP3nM/tracks?limit=100&offset=900" \
-H "Accept: application/json" -H "Content-Type: application/json" \
-H "Authorization: Bearer <TOKEN>" \
> /tmp/data.txt
```
2. Then run `jq .items[].track.id /tmp/data.txt` to view the list of track's ids

**Expected behavior**
There should be at least one null entry

### PR DESCRIPTION
**Pull request by marioortizmanero**
## Description

This fixes #279

## Dependencies 

None

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

The CI passes perfectly.

### CODE REVIEW

File: tests/test_with_oauth.rs [hunk 1]
```diff
@@ -385,13 +385,15 @@ async fn test_playback() {
...
     if let Some(backup) = &backup {
         let uri = backup.item.as_ref().map(|item| item.id());
-        let offset = None;
-        let device = backup.device.id.as_deref();
-        let position = backup.progress.map(|p| p.as_millis() as u32);
-        client
-            .start_uris_playback(uri, device, offset, position)
-            .await
-            .unwrap();
+        if let Some(uri) = uri {
```
Comment (ramsayleung) [inline tests/test_with_oauth.rs:388]: Is it necessary to extract the `uri` variable? It seems that `uri` will not be used later

**Comment by ramsayleung** [comment]
Everything else looks good to me.

**Review by ramsayleung — APPROVED** [review]

File: tests/test_with_oauth.rs (hunk 1 above)
Comment (marioortizmanero) [reply tests/test_with_oauth.rs:388]: It's used in `start_uris_playback`

**Comment by marioortizmanero** [comment]
Whoops, I have to add these changes to the changelog before this is merged.

**Comment by ramsayleung** [comment]
>  We should probably add a reminder for this in the PR template.

Great point!

**Comment by marioortizmanero** [comment]
Just updated the changelog, this is ready to be merged. I'll make a new PR with the reminder about the changelog.

**Pull request merged**